src/components/AnalysisPage/AnalysisPage.jsx

```diff
@@ -12,6 +12,7 @@ const AnalysisPage = () => {
   const [fileFor3dModel, setFileFor3dModel] = useState(null);
   const [partId, setPartId] = useState(null);
   const [geoData, setGeoData] = useState(null);
+  const [thickData, setThickData] = useState(null);
```

Comment:
These names could end up a little confusing since they are basically the same but geoData is just an object whereas thickness is an array of objects. IDK if just adding like listOfThicknessData or something would make it clearer

Reply:
I changed this to listOfThicknessData!

My bad for not commenting on these immediately after fixing them.